src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java

```diff
@@ -312,8 +314,8 @@ private PacketInterfaceTerminalUpdate updateList() {
                             .setReps(machine.getSelfRep(), machine.getDisplayRep());
                     tracked.put(machine, entry);
                     trackedById.put(entry.id, entry);
+                    visited.add(machine);
                 }
-                visited.add(machine);
```

Comment:
why is that necessary,  wouldn't 'continue' skip adding anyway?

Reply:
It it will produce unnecessary `remove` command better to do not try to remove element that are not present in terminal